### DISCUSSION TITLE
Can we consider making a minified version where we only list the best filters (most used / active)

### DISCUSSION THREAD

**Discussion by DevShubam** [collinbarrett/FilterLists#3449] [Ideas]
Filterlists.com is great, but it has **a lot** of lists, I think it would be beneficial to make a minified version. Would this be possible?

**Comment by collinbarrett** [comment]
Thanks. I agree that it is a lot and there are some great opportunities to improve filtering, etc. I'm not personally sure about a distinct "minified" version, per se. But, we do hope to implement at least some of the info mentioned in https://github.com/collinbarrett/FilterLists/issues/371 at some point which could really help users. I just personally have been lacking in time lately. Thanks for the suggestion!
